src/app/components/FrostedGlassPromo/__snapshots__/index.test.jsx.snap

```diff
@@ -119,63 +98,112 @@ exports[`Frosted Glass Promo when given props directly 1`] = `
   }
 }
 
+.emotion-15:focus {
+  -webkit-text-decoration: underline;
+  text-decoration: underline;
+}
+
+.emotion-17 {
+  display: none;
+}
+
+@supports (filter: blur(15px)) {
+  .emotion-17 {
+    display: block;
+    z-index: 1;
+    position: absolute;
+    bottom: 0;
+    top: -15px;
+    left: 0;
+    right: 0;
+    background: #202224 url('https://ichef.bbci.co.uk/news/400/cpsdevpb/DDFC/test/_63482865_orange2.jpg');
+    background-repeat: no-repeat;
+    -webkit-background-size: cover;
+    background-size: cover;
+    -webkit-background-position: bottom;
+    background-position: bottom;
+    -webkit-transform: scaleX(1.15) scaleY(-1.15);
+    -moz-transform: scaleX(1.15) scaleY(-1.15);
+    -ms-transform: scaleX(1.15) scaleY(-1.15);
+    transform: scaleX(1.15) scaleY(-1.15);
+    -webkit-filter: blur(15px);
+    filter: blur(15px);
+  }
+}
+
 <div>
-  <a
+  <div
     class="emotion-0 emotion-1"
-    href="/mundo/internacional-23038380"
   >
-    <div
+    <a
+      aria-hidden="true"
       class="emotion-2 emotion-3"
+      href="/mundo/internacional-23038380"
+      tabindex="-1"
+    />
+    <div
+      class="emotion-4 emotion-5"
       data-e2e="image-placeholder"
       style="padding-bottom: 52%;"
     >
       <img
         alt="Orange 2"
-        class="emotion-4 emotion-5 emotion-6"
+        class="emotion-6 emotion-7 emotion-8"
         src="https://ichef.bbci.co.uk/news/400/cpsdevpb/DDFC/test/_63482865_orange2.jpg"
         width="976"
       />
     </div>
     <div
-      class="emotion-7 emotion-8"
+      class="emotion-9 emotion-10"
     >
-      <div
-        class="emotion-9 emotion-10"
-      />
       <div
         class="emotion-11 emotion-12"
-      />
-      <div
-        class="emotion-13 emotion-14"
       >
-        <div
-          class="emotion-15 emotion-16"
+        <h3
+          class="emotion-13 emotion-14"
         >
-          Además de sus decenas de museos tradicionales, Ciudad de México es una galería al aire libre de arte callejero, en la que exponen artistas nacionales y extranjeros. Lo invitamos a un recorrido.
-        </div>
+          <a
+            class="emotion-15 emotion-16"
+            href="/mundo/internacional-23038380"
+          >
+            Además de sus decenas de museos tradicionales, Ciudad de México es una galería al aire libre de arte callejero, en la que exponen artistas nacionales y extranjeros. Lo invitamos a un recorrido.
+          </a>
+        </h3>
         4 mayo 2021
```

Comment:
Missing time tag?

Reply:
The component is set up so that we're not always assuming the footer will be a date.  The way it's being used on the page does have a time element in this place, though: https://github.com/bbc/simorgh/blob/801985b1efa1162255e4ec965b496360fca5fa52/src/app/components/FrostedGlassPromo/__snapshots__/index.test.jsx.snap#L388